projects/storefrontlib/src/cms-components/configurator/commons/service/config-utils.service.ts

```diff
@@ -10,7 +11,10 @@ import { map, take } from 'rxjs/operators';
   providedIn: 'root',
 })
 export class ConfigUtilsService {
```

Comment:
This generic config util service might be a good fit this kind of method.
Do you anticipate some other UI-related util methods in the future? If yes, maybe we can create a dedicated service in that case.

Reply:
I can't judge at the moment whether there will be other UI related stuff.